databuilder/tests/unit/extractor/test_eventbridge_extractor.py

```diff
@@ -276,48 +359,17 @@ def test_extraction_with_multiple_result(self) -> None:
             extractor = EventBridgeExtractor()
             extractor.init(self.conf)
 
-            expected = TableMetadata(
-                "eventbridge",
-                cluster_name,
-                test_schema_openapi_3["info"]["title"],
-                registry_name,
-                None,
-                [
-                    ColumnMetadata("AWSEvent", None, openapi_3_aws_event_type, 0),
-                    ColumnMetadata(
-                        "OrderConfirmed", None, openapi_3_order_confirmed_type, 1
-                    ),
-                    ColumnMetadata(
-                        "Customer", "customer description", openapi_3_customer_type, 2
-                    ),
-                    ColumnMetadata("Item", None, openapi_3_item_type, 3),
-                ],
-                False,
-            )
-            self.assertEqual(expected.__repr__(), extractor.extract().__repr__())
-
-            expected = TableMetadata(
-                "eventbridge",
-                cluster_name,
-                test_schema_json_draft_4["title"],
-                registry_name,
-                test_schema_json_draft_4["description"],
-                [
-                    ColumnMetadata("version", "version description", "string", 0),
-                    ColumnMetadata("id", None, "string", 1),
-                    ColumnMetadata("detail-type", None, "string", 2),
-                    ColumnMetadata("source", None, "string", 3),
-                    ColumnMetadata("account", None, "string", 4),
-                    ColumnMetadata("time", None, "string", 5),
-                    ColumnMetadata("region", None, "string", 6),
-                    ColumnMetadata("resources", None, "array<string>", 7),
-                    ColumnMetadata("detail", None, json_draft_4_booking_done_type, 8),
-                ],
-                False,
-            )
-            self.assertEqual(expected.__repr__(), extractor.extract().__repr__())
+            for expected_schema in expected_openapi_3_tables:
+                self.assertEqual(
+                    expected_schema.__repr__(), extractor.extract().__repr__()
+                )
 
+            for expected_table in expected_json_draft_4_tables:
+                self.assertEqual(
+                    expected_table.__repr__(), extractor.extract().__repr__()
+                )
             self.assertIsNone(extractor.extract())
+
```

Comment:
could you add a test that validates your latest version of the schema extractor?